### PR TITLE
Cask repair update dbeaver enterprise

### PR DESCRIPTION
**Pull request by howie**
##### Instructions
- Look for and complete the section relevant to your submission. Delete the others, including these `Instructions`.
- `{{cask_file}}` represents the cask file you’re submitting/editing (if applicable).
- If there’s a checkbox you can’t complete for any reason, that’s OK. Just explain in detail why you weren’t able to do so.
#### Editing an existing cask
- [ ] Commit message includes cask’s name (and new version, if applicable).
- [ ] `brew cask audit --download {{cask_file}}` is error-free.
- [ ] `brew cask style --fix {{cask_file}}` left no offenses.
#### Adding a new cask
- [ ] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [ ] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [ ] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [ ] Commit message includes cask’s name.
- [ ] `brew cask audit --download {{cask_file}}` is error-free.
- [ ] `brew cask style --fix {{cask_file}}` left no offenses.
- [ ] `brew cask install {{cask_file}}` worked successfully.
- [ ] `brew cask uninstall {{cask_file}}` worked successfully.
### Changes to the core
- [ ] Followed [hacking.md](https://github.com/caskroom/homebrew-cask/blob/master/doc/development/hacking.md).

### CODE REVIEW

**Comment by vitorgalvao** [comment]
In the future, please submit separate pull requests for each cask, as more often than not this type of pull request brings problems with one or more of them, and it halts the inclusion of the other ones.

You need to fill the template.

File: Casks/tableau-public.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 cask 'tableau-public' do
-  version '9.3.0'
+  version '9.3'
   sha256 '73f82fb7eac3f99a52ed53f48457b031db2d38b9cf1a30462dffa655a4618012'
 
-  url "https://downloads.tableau.com/public/TableauPublic-#{version.dots_to_hyphens}.dmg"
+  url 'https://public.tableau.com/s/download/public/mac'
```
Comment (vitorgalvao) [inline Casks/tableau-public.rb]: Leave this `url` be. It still works, and is better.

**Comment by adidalal** [comment]
Fixed in https://github.com/caskroom/homebrew-cask/pull/21343

**Comment by howie** [comment]
HI because I still have this problem https://github.com/vitorgalvao/tiny-scripts/issues/20#issuecomment-219218275

**Pull request closed without merging**